scripts/chroot_reconst.sh

```diff
@@ -94,16 +90,18 @@ dpkg -i systemlock_0.1-1_all.deb
 # /usr/share/applications/lxqt-about.desktop
 # or ~/.local/usr/share override (in skel!)
 # set the option NoDisplay=true
-cat /usr/share/applications/lxqt-about.desktop
```

Comment:
This was used for removing the about lxqt entry in the menu,
Can be removed

Reply:
yes